src/main/scala/org/hammerlab/guacamole/DistributedUtil.scala

```diff
@@ -336,8 +350,11 @@ object DistributedUtil extends Logging {
     readsRDDs: PerSample[RDD[MappedRead]],
     lociPartitions: LociMap[Long],
     skipEmpty: Boolean,
-    function: PerSample[Pileup] => Iterator[T],
+    function: (PerSample[Pileup], ContigSequence) => Iterator[T],
     reference: ReferenceGenome): RDD[T] = {
+
+    val referenceBC = readsRDDs.head.context.broadcast(reference)
```

Comment:
The pileup function we're in here is `pileupFlatMapMultipleRDDs`, which is called once in the codebase, by `SomaticJointCaller`, which calls it once per Spark app run (it's also called three times in DistributedUtilSuite).

Per [my comment above](https://github.com/hammerlab/guacamole/pull/417/files#r58210739), I don't agree that this is a problem, but if you are concerned with optimizing those tests or for future callers that might make two pileupFlatmap passes over a dataset in one run, we can discuss making this a wrapper for a version that takes a Broadcast as a parameter.